erp-client/src/app/components/award/award.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="container">
```

Comment:
Nothing special here -- just copied the employee component to see all the Award JSON.